timelock-api/src/main/conjure/timelock-history.yml

```diff
@@ -0,0 +1,49 @@
+types:
+  imports:
+    PaxosValue:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosValue
+    PaxosAcceptorState:
```

Comment:
This is the class we got from `leader-election-impl`, so we'll probably need an adapter. It looks like that wraps proposal IDs and Values, which are from `leader-election-api`, and that seems OK.

Maybe we can look at whether this fits in the timelock corruption project?